build.gradle

```diff
@@ -87,7 +87,7 @@ testClusters.javaRestTest {
 
 allprojects {
     group = 'com.amazon.opendistroforelasticsearch'
-    version = "${opendistroVersion}.0"
+    version = "${opendistroVersion}"
```

Comment:
The last digit is intended for plugins to have their own minor version.
Could you help me understand why we'd like to remove it?

Reply:
Hmm, it was generating an -alpha1.0 build. I can track where it's used incorrectly or we can leave it. WDYT?

Reply:
Yeah because `-alpha1` was appended to the `gradle.properties` file. I would prefer to stick with 1.0.0.0 as we have decided.

Reply:
Same here as common-utils, lets have the 4th digit for the minor version.

Reply:
reverting